gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientWriteChannel.java

```diff
@@ -46,18 +45,18 @@ class GoogleCloudStorageClientWriteChannel extends AbstractGoogleAsyncWriteChann
   private final StorageResourceId resourceId;
   private WriteChannel writeChannel;
   private boolean uploadSucceeded = false;
-  // TODO: not supported as of now
-  // private final String requesterPaysProject;
 
   public GoogleCloudStorageClientWriteChannel(
       Storage storage,
       GoogleCloudStorageOptions storageOptions,
       StorageResourceId resourceId,
       CreateObjectOptions createOptions,
+      boolean requesterPays,
```

Comment:
consider renaming this to something like requesterPaysEnabled or something similar